fab/environments.yml

```diff
@@ -54,6 +54,58 @@ softlayer:
       async_restore_queue:
         concurrency: 4
       flower: {}
+  pillows:
+    '*':
+      AppDbChangeFeedPillow:
+        num_processes: 1
+      ApplicationBlobDeletionPillow:
+        num_processes: 1
+      ApplicationToElasticsearchPillow:
+        num_processes: 1
+      BlobDeletionPillow:
+        num_processes: 1
+      CacheInvalidatePillow:
+        num_processes: 1
+      CareBiharFluffPillow:
+        num_processes: 1
+      CaseSearchToElasticsearchPillow:
+        num_processes: 1
+      CaseToElasticsearchPillow:
+        num_processes: 1
+      DefaultChangeFeedPillow:
+        num_processes: 1
+      DomainDbKafkaPillow:
+        num_processes: 1
+      FormSubmissionMetadataTrackerPillow:
+        num_processes: 1
+      GroupPillow:
+        num_processes: 1
+      GroupToUserPillow:
+        num_processes: 1
+      kafka-ucr-main:
+        num_processes: 1
+      kafka-ucr-static:
+        num_processes: 1
+      KafkaDomainPillow:
+        num_processes: 1
+      LedgerToElasticsearchPillow:
+        num_processes: 1
+      ReportCaseToElasticsearchPillow:
+        num_processes: 1
+      ReportXFormToElasticsearchPillow:
+        num_processes: 1
+      SqlSMSPillow:
+        num_processes: 1
+      UnknownUsersPillow:
+        num_processes: 1
+      UserCacheInvalidatePillow:
+        num_processes: 1
+      UserGroupsDbKafkaPillow:
+        num_processes: 1
+      UserPillow:
+        num_processes: 1
+      XFormToElasticsearchPillow:
+        num_processes: 1
```

Comment:
Yeah. I had the same conflicted feelings about that, but need to get this out before pillow partitioning. I would be pro a refactor for both celery and pillows that enables including and excluding, but I think it's good that we're using the same type of config now